Sources/KlaviyoForms/InAppForms/IAFWebViewModel.swift

```diff
@@ -102,6 +109,8 @@ class IAFWebViewModel: KlaviyoWebViewModeling {
         loadScripts?.insert(sdkNameWKScript)
         loadScripts?.insert(sdkVersionWKScript)
         loadScripts?.insert(handshakeWKScript)
+        guard let dataEnvironmentWKScript else { return }
+        loadScripts?.insert(dataEnvironmentWKScript)
```

Comment:
```suggestion
        if let dataEnvironmentWKScript { 
            loadScripts?.insert(dataEnvironmentWKScript)
        }
```
I think it's a bit cleaner this way

Reply:
> if we were to add a new script below line 113

yeah that totally makes sense! guess I was implicitly enforcing that by putting it at the end but that doesn't necessarily safeguard it for any future additions/changes so I agree this more explicit conditional insert is better and clearer (probably was my personal bias of trying to keep it all as one indent level lol)